libs/client/client.ts

```diff
@@ -3,13 +3,21 @@ import { Commitment, TransferReponseData } from "libs/types";
 import path from "path";
 import { parentLogger } from "../utils";
 import type { NightfallZkpKeys } from "../nightfall/types";
-import { NightfallRecipientData } from "libs/transactions/types";
-// import type { Token } from "../tokens";
+import type { NightfallRecipientData } from "libs/transactions/types";
+import { NightfallSdkError } from "../utils/error";
 
 const logger = parentLogger.child({
   name: path.relative(process.cwd(), __filename),
 });
 
+axios.interceptors.response.use(
```

Comment:
If we had this somewhere in a separate file under the `utils` folder it would be good in the future, so that we wouldn't need to repeat this code everywhere it is needed.

Reply:
I'll add this in upcoming #70